tests/smoke/environment.bats

```diff
@@ -28,6 +28,10 @@
     l0 environment create --os windows test4
 }
 
+@test "environment create test5" {
```

Comment:
Not sure if we should add a service or task to the environment to test the recursive delete, breaks patterns of bats.

Reply:
ah good call. I can see an argument for both, I personally like the way you've done it here because it keeps the environment "suite" of tests fast.